src/styles/theming/common/switch/_sap_fiori.scss

```diff
@@ -70,4 +70,13 @@
   --fdSwitch_Compact_Semantic_Handle_Offset: -1.9375rem;
   --fdSwitch_Compact_Semantic_Active_Handle_Offset: 0.0625rem;
   --fdSwitch_Compact_Semantic_Active_Handle_Offset_Rtl: var(--fdSwitch_Active_Handle_Offset);
+
+  // Dynamic Page Layout
```

Comment:
It's probably shouldn't be here, in switch styles.

Reply:
oh, nice catch! I didn't see the `switch` folder, just the `common` folder.  :D